### ISSUE TITLE
preview.js without eject

### ISSUE THREAD

**Issue by sebalaini-itm** [nuxt-community/storybook#229]
we switched from storybook to nuxt storybook after some upgrade in our dependencies where the config was starting to become a bit too difficult to manage.
in our old `preview.js` we had to use some helpers functions in  it to make our setup working, with nuxt storybook seems that everything that's in the `plugins` array of the `nuxt.config` it's loaded with the exception of a `mixin` function, is there a way to load that `mixin` function into nuxt storybook without need to eject it?

`translation.js`
```
import Vue from 'vue'

export const translations = (words) => {
  Vue.$words = words

  Vue.mixin({
    methods: {
      lang (key) {
        ....
  })
}

export default (context, inject) => {
  inject('translations', translations)
}
```

`nuxt.config.js`
```
  plugins: [
    '~/helpers/plugins/translations.js',
    ....
  ],
```

old `.storybook/preview.js`
```
import { translations } from '~/helpers/plugins/translations'

translations({
  backtotop: 'Back to top',
  readMore: 'Read more',
 ...
})
```

and here the error
![image](https://user-images.githubusercontent.com/61748264/107969214-60d93000-6fa7-11eb-9700-6e6d70d30462.png)

**Comment by loilo** [comment]
I second this. For us, it is including storybook-specific styles via `preview.js`.

**Comment by sebalaini-itm** [comment]
thanks for the answer @farnabaz , in the end I managed to get it work like this,

in the 1st story, I've added this code and seems to work, maybe not the best way but it does the job

```
// this is needed only once for all the storybook stories
import { translations } from '~/helpers/plugins/translations'
import testTranslations from '~/test-data/translations'

translations(testTranslations)
```

**Comment by loilo** [comment]
> @loilo Could you explain your idea?

I wanted to inject styles into Storybook that are not part of the original application, mostly because we use Storybook as kind of a hybrid between design system and component documentation (which we include alongside the components themselves).

For the documentation, we need some styles (e.g. for code blocks) that are not used in the app itself.

We currently work our way around this by using a special `<StorybookWrapper>` component (which includes these styles) as a Storybook decorator, but that doesn't feel quite right because it lives alongside our "normal" app components and I'm not sure if it ends up in the app's production code (where it is not needed at all), although it currently doesn't seem to.

**Comment by sebalaini-itm** [comment]
closing as I fixed it in my own way but would be nice to add the possibility to merge `.preview.js` configs directly from the `nuxt.config` section.
